### ISSUE TITLE
Signature extraction failed on a video

### ISSUE THREAD

**Issue by goblin** [rg3/youtube-dl#11693]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.01.10*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.01.10**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
**NOTE:** This is possibly a duplicate of https://github.com/rg3/youtube-dl/issues/11692 but it's hard to tell as the whole backtrace is put in the subject

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

Full error:
```
$ ./youtube-dl --verbose https://www.youtube.com/watch?v=x2_j6kMg1co
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://www.youtube.com/watch?v=x2_j6kMg1co']
WARNING: Assuming --restrict-filenames since file system encoding cannot encode all characters. Set the LC_ALL environment variable to fix this.
[debug] Encodings: locale ANSI_X3.4-1968, fs ascii, out ANSI_X3.4-1968, pref ANSI_X3.4-1968
[debug] youtube-dl version 2017.01.10
[debug] Python version 3.5.2 - Linux-4.5.0-gblARCH-x86_64-with-arch-Arch-Linux
[debug] exe versions: ffmpeg 3.2.2, ffprobe 3.2.2, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] x2_j6kMg1co: Downloading webpage
[youtube] x2_j6kMg1co: Downloading video info webpage
[youtube] x2_j6kMg1co: Extracting video information
[youtube] {22} signature length 44.40, html5 player en_US-vflC029_L
[youtube] x2_j6kMg1co: Downloading player /yts/jsbin/player-en_US-vflC029_L/base.js
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1011, in _decrypt_signature
    video_id, player_url, s
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 925, in _extract_signature_function
    errnote='Download of %s failed' % player_url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 519, in _download_webpage
    res = self._download_webpage_handle(url_or_request, video_id, note, errnote, fatal, encoding=encoding, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 426, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 406, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 2000, in urlopen
    req = sanitized_Request(req)
  File "./youtube-dl/youtube_dl/utils.py", line 518, in sanitized_Request
    return compat_urllib_request.Request(sanitize_url(url), *args, **kwargs)
  File "/usr/lib/python3.5/urllib/request.py", line 269, in __init__
    self.full_url = url
  File "/usr/lib/python3.5/urllib/request.py", line 295, in full_url
    self._parse()
  File "/usr/lib/python3.5/urllib/request.py", line 324, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'
 (caused by ValueError("unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1011, in _decrypt_signature
    video_id, player_url, s
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 925, in _extract_signature_function
    errnote='Download of %s failed' % player_url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 519, in _download_webpage
    res = self._download_webpage_handle(url_or_request, video_id, note, errnote, fatal, encoding=encoding, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 426, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 406, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 2000, in urlopen
    req = sanitized_Request(req)
  File "./youtube-dl/youtube_dl/utils.py", line 518, in sanitized_Request
    return compat_urllib_request.Request(sanitize_url(url), *args, **kwargs)
  File "/usr/lib/python3.5/urllib/request.py", line 269, in __init__
    self.full_url = url
  File "/usr/lib/python3.5/urllib/request.py", line 295, in full_url
    self._parse()
  File "/usr/lib/python3.5/urllib/request.py", line 324, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'
Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1011, in _decrypt_signature
    video_id, player_url, s
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 925, in _extract_signature_function
    errnote='Download of %s failed' % player_url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 519, in _download_webpage
    res = self._download_webpage_handle(url_or_request, video_id, note, errnote, fatal, encoding=encoding, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 426, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 406, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 2000, in urlopen
    req = sanitized_Request(req)
  File "./youtube-dl/youtube_dl/utils.py", line 518, in sanitized_Request
    return compat_urllib_request.Request(sanitize_url(url), *args, **kwargs)
  File "/usr/lib/python3.5/urllib/request.py", line 269, in __init__
    self.full_url = url
  File "/usr/lib/python3.5/urllib/request.py", line 295, in full_url
    self._parse()
  File "/usr/lib/python3.5/urllib/request.py", line 324, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 694, in extract_info
    ie_result = ie.extract(url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 359, in extract
    return self._real_extract(url)
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1590, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1021, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
youtube_dl.utils.ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 1011, in _decrypt_signature
    video_id, player_url, s
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 925, in _extract_signature_function
    errnote='Download of %s failed' % player_url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 519, in _download_webpage
    res = self._download_webpage_handle(url_or_request, video_id, note, errnote, fatal, encoding=encoding, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 426, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 406, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 2000, in urlopen
    req = sanitized_Request(req)
  File "./youtube-dl/youtube_dl/utils.py", line 518, in sanitized_Request
    return compat_urllib_request.Request(sanitize_url(url), *args, **kwargs)
  File "/usr/lib/python3.5/urllib/request.py", line 269, in __init__
    self.full_url = url
  File "/usr/lib/python3.5/urllib/request.py", line 295, in full_url
    self._parse()
  File "/usr/lib/python3.5/urllib/request.py", line 324, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'
 (caused by ValueError("unknown url type: '/yts/jsbin/player-en_US-vflC029_L/base.js'",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


```

---

### Description of your *issue*, suggested solution and other information

I tried getting the video at https://www.youtube.com/watch?v=x2_j6kMg1co and encountered the problem as seen in the stacktrace above. The program asked me to report this, so here I am, reporting.

Thanks.

**Comment by goblin** [comment]
Hmm OK I've just tried downloading this again and this time it worked... Must have been a temporary problem...

**Comment by paddatrapper** [comment]
I am currently getting this issue. Retrying does not fix it. Also running latest version from within a python script

**Comment by paddatrapper** [comment]
Though it seems I'm getting a different stack trace:
````
[youtube] d2UZlwTDGbY: Downloading webpage
[youtube] d2UZlwTDGbY: Downloading video info webpage
[youtube] d2UZlwTDGbY: Extracting video information
[youtube] d2UZlwTDGbY: Downloading js player en_US-vflC029_L
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'Ai'
 (caused by KeyError('Ai',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'Ai'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 682, in extract_info
    ie_result = ie.extract(url)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/common.py", line 347, in extract
    return self._real_extract(url)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1578, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1015, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
youtube_dl.utils.ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'Ai'
 (caused by KeyError('Ai',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./download-songs.py", line 25, in <module>
    ydl.download([song])
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 1775, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 697, in extract_info
    self.report_error(compat_str(e), e.format_traceback())
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 556, in report_error
    self.trouble(error_message, tb)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 526, in trouble
    raise DownloadError(message, exc_info)
youtube_dl.utils.DownloadError: ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/home/kyle/.local/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'Ai'
 (caused by KeyError('Ai',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
````

**Comment by dstftw** [comment]
Post the complete log with `'verbose': True`.

**Comment by paddatrapper** [comment]
Noticed I had an old version in my path that was taking priority. Removing it (and so actually using the latest version) fixed it

**Comment by eelleet** [comment]
i had this issue but updating to the latest version resolved it for me. It seems the version i got from my package manager is VERY outdated ( 2016.02.22). After uninstalling and manually reinstalling the version number was updated to 2017.1.16 

-hope this helps 

error-output 
youtube-dl --verbose --extract-audio --audio-format mp3 https://www.youtube.com/watch?v=k0bQuzp9Iss&t=2413s
[1] 31881
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'--extract-audio', u'--audio-format', u'mp3', u'https://www.youtube.com/watch?v=k0bQuzp9Iss']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.02.22
[debug] Python version 2.7.12 - Linux-4.4.0-57-generic-x86_64-with-Ubuntu-16.04-xenial
[debug] exe versions: ffmpeg 2.8.10-0ubuntu0.16.04.1, ffprobe 2.8.10-0ubuntu0.16.04.1, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] k0bQuzp9Iss: Downloading webpage
[youtube] k0bQuzp9Iss: Downloading video info webpage
[youtube] k0bQuzp9Iss: Extracting video information
[youtube] {22} signature length 41.42, html5 player en_US-vflR62D9G
[youtube] k0bQuzp9Iss: Downloading player https://s.ytimg.com/yts/jsbin/player-en_US-vflR62D9G/base.js
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 905, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 820, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 883, in _parse_sig_js
    initial_function = jsi.extract_function(funcname)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 241, in extract_function
    raise ExtractorError('Could not find JS function %r' % funcname)
ExtractorError: Could not find JS function u'Gi'; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
 (caused by ExtractorError(u"Could not find JS function u'Gi'; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 905, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 820, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 883, in _parse_sig_js
    initial_function = jsi.extract_function(funcname)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 241, in extract_function
    raise ExtractorError('Could not find JS function %r' % funcname)
ExtractorError: Could not find JS function u'Gi'; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/YoutubeDL.py", line 666, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 316, in extract
    return self._real_extract(url)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 1407, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 915, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 905, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 820, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 883, in _parse_sig_js
    initial_function = jsi.extract_function(funcname)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/jsinterp.py", line 241, in extract_function
    raise ExtractorError('Could not find JS function %r' % funcname)
ExtractorError: Could not find JS function u'Gi'; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
 (caused by ExtractorError(u"Could not find JS function u'Gi'; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


[1]+  Exit 1                  youtube-dl --verbose --extract-audio --audio-format mp3 https://www.youtube.com/watch?v=k0bQuzp9Iss

**Comment by paddatrapper** [comment]
I also found manually updating worked. The problem I didn't notice was
that I had version 2016.01.22 installed in `~/.local`, while the package
manager was installing the new package in `/usr/share`. Removing the
version in `~/.local` fixed the issue

**Comment by yan12125** [comment]
This should have been fixed in version 2017.01.31 or newer. Feel free to open new issues if errrors re-emerge.
